src/ethereum/api/Bridge.ts

```diff
@@ -90,16 +93,21 @@ export class Bridge {
 
     for (const urn in itemsByURN) {
       const item = itemsByURN[urn]
+      const itemCuration = itemCurations.find((ic) => ic.item_id === item.id)
```

Comment:
I don't want to add further complexity here, but maybe we can index the itemCurations too (as we do collections) so we don't have to iterate multiple times. This is what I have in mind

```ts
    const dbTPCollectionsIndex = this.indexById(dbTPCollections)
    const itemCurationsIndex = this.indexBy(itemCurations, 'item_id')
```

```ts
indexById(list) {
  return this.indexById(list, 'id')
}
indexBy(list, prop) {}
```

What do you think? might be overkill

Reply:
Yeah, that's a good idea! Implemented :)